packages/skeleton/src/lib/utilities/Modal/Modal.svelte

```diff
@@ -183,7 +183,11 @@
 
 	function onPromptSubmit(event: SvelteEvent<SubmitEvent, HTMLFormElement>): void {
 		event.preventDefault();
-		if ($modalStore[0].response) $modalStore[0].response(promptValue);
+		if ($modalStore[0].response) {
+			if ($modalStore[0].valueAttr !== undefined && 'type' in $modalStore[0].valueAttr && $modalStore[0].valueAttr.type === 'number')
+				$modalStore[0].response(parseInt(promptValue));
```

Comment:
Why not Number constructor 🤔? I generally think it's better, not that it would make a huge performance leap in this context.
```ts
$modalStore[0].response(Number(promptValue));
```

Feel free to correct me if I'm wrong.

Reply:
@phamduylong It is not better (performance depends on the string we are trying to parse), it just behaves differently when parsing, and to be honest both behaviors are totally Ok in our case, so I didn't think about it and would leave it as is if there is not a good argument behind the change.
but thanks for the suggestion and feel free to follow up if you have a better reason for the change :smile: